pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>ai.rev.speechtotext</groupId>
     <artifactId>revai-java-sdk-speechtotext</artifactId>
-    <version>1.0.0</version>
+    <version>1.1.0</version>
```

Comment:
Please double check with @Chanc3 that this is the proper way to bump the version, but seems reasonable to me.

Reply:
Talked to @Chanc3 , this is the correct way to bump the version.